tests/pytorch/nightly/hf-glue.libsonnet

```diff
@@ -177,16 +222,38 @@ local utils = import 'templates/utils.libsonnet';
       },
     },
   },
+  local tpuVm = common.PyTorchTpuVmMixin {
+    tpuSettings+: {
+      tpuVmExports+: |||
+        export XLA_USE_BF16=$(XLA_USE_BF16)
+      |||,
+      tpuVmExtraSetup: |||
+        pip install pathlib tensorboardX google-cloud-storage
+        git clone --recursive https://github.com/pytorch-tpu/examples.git tpu-examples/
```

Comment:
You can probbaly remove these three lines that install fairseq and add it to the PATH unless HF depends on fairseq.

Reply:
removed